python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -67,6 +75,7 @@ def dbt_assets(
     backfill_policy: Optional[BackfillPolicy] = None,
     op_tags: Optional[Mapping[str, Any]] = None,
     required_resource_keys: Optional[Set[str]] = None,
+    project: Optional[DbtProject] = None,
```

Comment:
This should be in a separate PR, but we should think about enforcing the fork in the road here: 

A user should either provide `manifest` or `project`, but not both.

But we should also be cognizant of this causing an instance of https://github.com/dagster-io/internal/discussions/9694.

Reply:
Yeah, I do think the "only one of manifest/project" is unfortunate. I'm not sure if there's a better way around it though since we can't alter the `manifest` param signature.